account_analytic_document_date/models/account_move.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 (APSL - Nagarro) Miquel Pascual, Bernat Obrador
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    document_date = fields.Date()
```

Comment:
If it's a date only for analytic, it shouldn't be called `document_date`, but `analytic_document_date` or `analytic_date`.

Reply:
But it creates confusion on users thinking that date is going to be used for example for financial reports. It should be renamed to `analytic_document_date` IMO.